scripts/repl.js

```diff
@@ -31,8 +31,10 @@
   var getQuery = function() {
     var query = window.location.hash.replace(/^\#\?/, '');
     return _.transform(query ? query.split('&') : null, function(result, val) {
-      val = val.split('=');
-      result[val[0]] = decodeURIComponent('' + val[1]);
+      var equalsIndex = val.indexOf('=');
+      var name = val.substring(0, equalsIndex);
+      var value = val.substring(equalsIndex + 2);
```

Comment:
@sparty02 mind opening that PR now?

@fabiomcosta Sorry you didn't get a commit in! We welcome any and all help, there's a bunch of issues open right now if you want to take any of them https://github.com/6to5/6to5.github.io/issues